quickwit/quickwit-ingest/src/ingest_v2/router.rs

```diff
@@ -134,6 +134,7 @@ impl IngestRouter {
             state_guard.shard_table.insert_shards(
                 index_id,
                 subresponse.source_id,
+                self.self_node_id.to_string(),
```

Comment:
The `self_node_id` should be set on the shard table when it is instantiated in `IngestRouter::new`.